docs/tracking-javascript-guide.md

```diff
@@ -693,6 +693,52 @@ _paq.push(['trackPageView']);
 
 ## Multiple Piwik trackers
 
+### If you use Piwik 2.16.2 or later
+
+It is possible to track your analytics data into multiple separate Piwik trackers. You may record your data either in the same Piwik server (into a different website ID) or you may record a duplicate of your data into another Piwik server altogether. 
+
+The following example shows how to use `addTracker` to store all of your data in another Piwik server hosted at `analytics.example.com`:
```

Comment:
"how to use `addTracker` " => "how to use `addTracker` method to store all of your data into another"